chef/cookbooks/provisioner/templates/default/zzz-prompt.sh.erb

```diff
@@ -7,7 +7,12 @@ fi
 # Only set the crazy prompt on terminals that support colors
 # it looks horrible on monochromatic terminals
 
-if [ $(tput colors) -ge 8 ]; then
+# Make sure that colors has a value even when tput does not recognize the
+# terminal type.
+colors=$(tput colors 2> /dev/null)
```

Comment:
ugh, one ugly problem here is.. this is a global variable :-( can we do this without a named variable? imho there is no need to do that in multiple expressions